src/hooks/useApiCall.js

```diff
@@ -72,54 +68,25 @@ export default function useAPICall (callType, params) {
 
   async function getItemInfo (itemKey, gender) {
     try {
-      const request = createRequestObject('browse', {
-        search: itemKey,
-        gender
+      const request = createRequestObject('iteminfo', {
+        item_key: itemKey,
+        gender,
       })
 
       const response = await fetch(request.url, request.headers)
       if (!response.ok) throw new Error()
 
-      const results = await response.json()
-      const itemInfo = extractItemInfo(results, itemKey)
-
-      if (!itemInfo) { throw Error() }
+      const itemInfo = await response.json()
+      if (!itemInfo) throw new Error()
 
-      return {
-        sku: itemInfo.sku,
-        modelName: itemInfo.model,
-        image: itemInfo.image,
-        url: itemInfo.url,
-        urlKey: itemInfo.urlKey
-      }
+      return itemInfo
+    
     } catch (e) {
```

Comment:
If we catch `(e)` we should do something with it, why don't we just `catch {` here?